scripts/deploy_on_odh.sh

```
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR="$(dirname "$(realpath "$BASH_SOURCE")")"
```

Shellcheck (0.10.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)